lib/core/include/irods/irods_logger.hpp

```diff
@@ -33,126 +34,846 @@
 #include <algorithm>
 #include <iomanip>
 #include <sstream>
+#include <concepts>
 
-namespace irods::experimental
+/// Defines all things related to the logging API.
+///
+/// \since 4.3.0
+namespace irods::experimental::log
 {
-    class log
```

Comment:
f3db4cdb8d7069862b71e3ac96bb72a0964b0c61

Please add a note to the commit message about this change as it is not purely merging the two files.

Reply:
done